hooks/PythNetwork/useScalingFactor.ts

```diff
@@ -0,0 +1,33 @@
+import NodeWallet from "@coral-xyz/anchor/dist/cjs/nodewallet";
+import { determineVotingPowerType } from "@hooks/queries/governancePower";
+import { useRealmQuery } from "@hooks/queries/realm";
+import { PythClient } from "@pythnetwork/staking";
+import { useConnection } from "@solana/wallet-adapter-react";
+import { useAsync } from "react-async-hook";
+
+
+export default function useScalingFactor() : number | undefined {
+    const realm = useRealmQuery().data?.result
+    const {connection} = useConnection()
+    const { result: plugin } = useAsync(
```

Comment:
Yes, I agree this would calculate scaling factor for each proposal. I think we should use useQuery instead. That will help us calculate scaling factor only once and then reuse it everywhere.

Reply:
Done